core/templates/pages/collection-editor-page/templates/collection-editor-pre-publish-modal.controller.spec.ts

```diff
@@ -88,21 +89,24 @@ describe('Collection Editor Pre Publish Modal Controller', function() {
       expect(ctrl.newCategory).toBe(collectionDict.category);
     });
 
-    it('should save a collection', function() {
-      ctrl.newTitle = 'New title';
-      ctrl.newObjective = 'New objective';
-      ctrl.newCategory = 'Algorithm';
-      expect(ctrl.isSavingAllowed()).toBe(true);
+    it('should save a collection when it has title, objective and category',
```

Comment:
```suggestion
    it('should allow saving a collection with a title, objective and category, and should save the collection',
```

Reply:
`save the collection` is being repetitive.